stable/airflow/README.md

```diff
@@ -356,6 +356,8 @@ The following table lists the configurable parameters of the Airflow chart and t
 | `web.resources`                          | custom resource configuration for web pod               | `{}`                      |
 | `web.initialStartupDelay`                | amount of time webserver pod should sleep before initializing webserver             | `60`  |
 | `web.initialDelaySeconds`                | initial delay on livenessprobe before checking if webserver is available    | `360` |
+| `web.secretsDir`                         | directory in which to mount secrets on webserver nodes  | /var/airflow/secrets      |
+| `web.secrets`                            | secrets to mount as volumes on webserver nodes          | []                        |
```

Comment:
Can you add a section in the README describing the usecase where you want a secret in your worker ?

Reply:
this is in webserver, workers already have this, so I am basically leveling the functionality

Reply:
Ok, just for my information, why is it needed in the web server ?

Reply:
https://airflow.apache.org/plugins.html plugins can run in webserver. They should be able to leverage secrets too

Reply:
@gsemet is this sufficient